openquake/calculators/classical.py

```diff
@@ -129,8 +161,7 @@ def __bool__(self):
         """
         True if the bounding box is non empty.
         """
-        return (self.min_dist is not None and self.west is not None and
-                self.south is not None)
+        return bool(self.min_dist and self.west and self.south)
```

Comment:
Please add a comment about the fact we are not checking also north and east here.